source/jormungandr/jormungandr/autocomplete/kraken.py

```diff
@@ -34,31 +34,14 @@
 from jormungandr.interfaces.v1.decorators import get_serializer
 from jormungandr.interfaces.v1.serializer import api
 from jormungandr.scenarios.utils import build_pagination, places_type
-from jormungandr.interfaces.v1.fields import (
-    NonNullList,
-    place,
-    NonNullNested,
-    PbField,
-    error,
-    feed_publisher,
-    disruption_marshaller,
-)
-from flask_restful import fields, abort
+from flask_restful import abort
 import navitiacommon.request_pb2 as request_pb2
 import navitiacommon.type_pb2 as type_pb2
 from jormungandr.utils import date_to_timestamp
 
 
-places = {
-    "places": NonNullList(NonNullNested(place)),
-    "error": PbField(error, attribute='error'),
-    "disruptions": fields.List(NonNullNested(disruption_marshaller), attribute="impacts"),
-    "feed_publishers": fields.List(NonNullNested(feed_publisher)),
-}
-
-
 class Kraken(AbstractAutocomplete):
-    @get_serializer(serpy=api.PlacesSerializer, marshall=places)
+    @get_serializer(serpy=api.PlacesSerializer)
```

Comment:
The function get_serializer() in decorators.py still has the parameter 'marshall'. If it isn't used anymore, it can be removed.

Reply:
I removed the `marshall` parameter, but I kept the functions